src/main/kotlin/com/zup/pagamento/repository/FormaPagamentoRepository.kt

```diff
@@ -1,7 +1,24 @@
 package com.zup.pagamento.repository
 
 import com.zup.pagamento.model.FormaPagamento
+import com.zup.pagamento.model.FormaPagamentoResponse
 import org.springframework.data.jpa.repository.JpaRepository
+import org.springframework.data.jpa.repository.Query
+import org.springframework.data.repository.query.Param
 
 interface FormaPagamentoRepository : JpaRepository<FormaPagamento, Long> {
+
+    @Query(nativeQuery = true,
+            value = "select fp.id, \n" +
```

Comment:
Aqui não rola o multiline => https://realkotlin.com/tutorials/2018-06-26-multiline-string-literals-in-kotlin/ ?

Reply:
Corrigido.